src/manifolds/Hyperbolic.jl

```diff
@@ -61,35 +49,39 @@ function check_manifold_point(M::Hyperbolic, p; kwargs...)
             "The point $(p) does not lie on $(M), since its size is not $(representation_size(M)).",
         )
     end
-    if !isapprox(minkowski_dot(p, p), -1.0; kwargs...)
+    if !isapprox(inner(M,p,p,p), -1.0; kwargs...)
```

Comment:
```suggestion
    if !isapprox(minkowski_dot(p, p), -1.0; kwargs...)
```
I think using `minkowski_dot` is clearer in this case since `p` does not represent a tangent vector.

Reply:
Similarly below.

Reply:
Yes, I thought about that, but then we again have two implementations of Minkowski-dot unless you also use that for the inner.

Reply:
Yes, `minkowski_dot` can also be used in `inner`.